apps/passwd.c

```diff
@@ -556,7 +556,7 @@ static char *shacrypt(const char *passwd, const char *magic, const char *salt)
     OPENSSL_strlcat(out_buf, "$", sizeof out_buf);
     if (rounds_custom) {
         char tmp_buf[7 + 9 + 1]; /* "rounds=999999999" */
```

Comment:
Aehm, the buffer is too short for 64bit integers. (9 is even too short for 32bit integers)